### DISCUSSION TITLE
change default port

### DISCUSSION THREAD

**Discussion by vipas84** [pocketbase/pocketbase#4839] [Q&A] [answered]
I'm using pocketbase as a framework and I need to change the default port (8090) for my binaries to a new one (8091).
is there any wat to so besides launching it each time with port argument?

**Comment by ganigeorgiev** [comment, marked as answer]
It is not clear what do you mean by _"besides launching it each time with port argument"_.

You can change the default PocketBase server port with the `--http` flag:

```sh
# run with --help to list all options
./pocketbase serve --http="127.0.0.1:8091"
```

If that's not what you are looking for please elaborate.

**Reply by vipas84** [reply]
thank you very much for your answer - it works great!
my question was related to override default port before binary compilation, but I guess its too much...

**Reply by npereira-jc** [reply]
any ideas on how to do this on docker?

**Reply by ganigeorgiev** [reply]
@npereira-jc You can pass it as part of the CMD instructions. You could find an example Dockerfile in https://pocketbase.io/docs/going-to-production/#using-docker.
